system/indy-node-tests/TestAdHocSuite.py

```diff
@@ -52,7 +51,8 @@ class TestAdHocSuite:
             assert res3['result']['seqNo'] is not None
 
             # check that pool is ok
-            await ensure_pool_is_in_sync(nodes_num=nodes_num)
+            await ensure_all_nodes_online(pool_handler, wallet_handler, trustee_did)
+            await ensure_ledgers_are_in_sync(pool_handler, wallet_handler, trustee_did)
             await ensure_state_root_hashes_are_in_sync(pool_handler, wallet_handler, trustee_did)
```

Comment:
I see these functions together quite often, why not create wrapper like `ensure_pool_is_okay`?

Reply:
Done.